src/sentry/static/sentry/app/views/projectDetail/projectLatestAlerts.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import {Location} from 'history';
+import pick from 'lodash/pick';
+import {withTheme} from 'emotion-theming';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import {Organization} from 'app/types';
+import {SectionHeading} from 'app/components/charts/styles';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import styled from 'app/styled';
+import {URL_PARAM} from 'app/constants/globalSelectionHeader';
+import {IconCheckmark, IconFire, IconWarning} from 'app/icons';
+import TimeSince from 'app/components/timeSince';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Theme} from 'app/utils/theme';
+import Link from 'app/components/links/link';
+import Placeholder from 'app/components/placeholder';
+import ButtonBar from 'app/components/buttonBar';
+import Button from 'app/components/button';
+import CreateAlertButton from 'app/components/createAlertButton';
+
+import {Incident, IncidentStatus} from '../alerts/types';
+
+const DOCS_URL = 'https://docs.sentry.io/workflow/alerts-notifications/alerts/';
+
+type Props = AsyncComponent['props'] & {
+  organization: Organization;
+  projectSlug: string;
+  location: Location;
+  theme: Theme;
+};
+
+type State = {
+  unresolvedAlerts: Incident[] | null;
+  resolvedAlerts: Incident[] | null;
+  hasAlertRule: boolean;
+} & AsyncComponent['state'];
+
+class ProjectLatestAlerts extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {location, organization} = this.props;
+
+    const query = {
+      ...pick(location.query, [...Object.values(URL_PARAM)]),
+      per_page: 3,
+    };
+
+    // we are listing 3 alerts total, first unresolved and then we fill with resolved
+    return [
+      [
+        'unresolvedAlerts',
+        `/organizations/${organization.slug}/incidents/`,
+        {query: {...query, status: IncidentStatus.OPENED}},
+      ],
+      [
+        'resolvedAlerts',
+        `/organizations/${organization.slug}/incidents/`,
+        {query: {...query, status: IncidentStatus.CLOSED}},
```

Comment:
Would that be as simple as adding status here?
https://github.com/getsentry/sentry/blob/4100d5025002601d9c22c3aec61dea17a3d36cf1/src%2Fsentry%2Fincidents%2Fendpoints%2Forganization_incident_index.py#L53

These are our statuses:
https://github.com/getsentry/sentry/blob/d11f482939633ab6527c189d6aef04636fb2974b/src%2Fsentry%2Fincidents%2Fmodels.py#L124-L128

We would probably want Critical > Warning > Open > Close.

Not sure what's the difference between Open and Critical|Warning.

Reply:
You could get Critical > Warning > Open if you did `order_by=-status` and excluded the closed status.

Reply:
We are also interested in closed ones. So if we get only one non-closed, we want to fill the rest (up to 3 in this case) with closed ones. How should we go about doing that?

Reply:
I am going to merge this as is so that we can play with it internally. 

We will need to add a few filtering options to that endpoint in the near future anyway, we can also tackle this sorting problem at the same time.